src/core/ext/filters/client_channel/backup_poller.cc

```diff
@@ -16,18 +16,19 @@
  *
  */
 
-#include <grpc/support/port_platform.h>
```

Comment:
This should always be the first header.

Reply:
Actually clang-format moves its location. Should I follow clang-format here?

Reply:
I think reverting your change here will pass the clang format check.

Reply:
To be clear, this is an auto change made by 'git clang-format', which unfortunately breaks the port platform check on github.

Reply:
`git clang-format`? I believe we should use `tools/distrib/clang_format_code.sh` to format the code.

The sanity check here should definitely override `git clang-format`.